### PR TITLE
2 Random fixes

### PR DESCRIPTION
**Pull request by mstratford**

### CODE REVIEW

**Comment by mstratford** [comment]
It didn't seem to be picking up the trackid field previously, so copied bits from elsewhere.

**Comment by mstratford** [comment]
Might be to do with how "Number" is the first col of rec_track, not trackid.

File: src/Classes/ServiceAPI/MyRadio_Track.php [hunk 1]
```diff
@@ -1289,11 +1289,11 @@ public function setIntro($duration)
     public static function getAllDigitised()
     {
         self::initDB();
-        $result = self::$db->fetchColumn(self::BASE_TRACK_SQL.' WHERE digitised=\'t\'');
```
Comment (LordAro) [inline src/Classes/ServiceAPI/MyRadio_Track.php]: BASE_TRACK_SQL appears to only be used on one other place - get rid of it?

File: src/Controllers/Mail/send.php [hunk 2]
```diff
@@ -27,14 +27,17 @@
...
         ]
     )
 )->addField(
     new MyRadioFormField(
         'body',
         MyRadioFormField::TYPE_BLOCKTEXT,
-        []
+        [
+            'label' => 'Message Body',
+        ]
```
Comment (LordAro) [inline src/Controllers/Mail/send.php]: doesn't really matter, but squash these 3 onto 1 line?

**Pull request merged**